pipelines/fbc-release/fbc-release.yaml

```diff
@@ -221,6 +221,26 @@ spec:
           value: "$(context.pipelineRun.uid)/data.json"
         - name: ocpVersion
           value: "$(tasks.get-ocp-version.results.stored-version)"
+    - name: check-fbc-packages
+      workspaces:
+        - name: data
+          workspace: release-workspace
+      taskRef:
+        resolver: "git"
+        params:
+          - name: url
+            value: $(params.taskGitUrl)
+          - name: revision
+            value: $(params.taskGitRevision)
+          - name: pathInRepo
+            value: tasks/check-fbc-packages/check-fbc-packages.yaml
+      params:
+        - name: dataPath
+          value: "$(context.pipelineRun.uid)/data.json"
```

Comment:
we replaced these in favor of collect-data task results (this and line 241)

Reply:
Got it. Fixed!